hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3GuardTool.java

```diff
@@ -434,7 +434,8 @@ public abstract int run(String[] args, PrintStream out) throws Exception,
         "\n" +
         "  URLs for Amazon DynamoDB are of the form dynamodb://TABLE_NAME.\n" +
         "  Specifying both the -" + REGION_FLAG + " option and an S3A path\n" +
-        "  is not supported.";
+        "  is not supported.\n"
+        + "To create a table with per-request billing, set the read and write capaciies to 0";
```

Comment:
typo: capaciies -> capacities

Reply:
fixed; also splitting the line in both the IDE and in the printed text